Makefile

```diff
@@ -123,12 +123,26 @@ endif
 
 test-backend-unit:
 	cd lumigator/python/mzai/backend/; \
-	SQLALCHEMY_DATABASE_URL=sqlite:////tmp/local.db uv run pytest -o python_files="backend/tests/unit/*/test_*.py"
+	S3_BUCKET=lumigator-storage \
+	RAY_HEAD_NODE_HOST=localhost \
+	RAY_DASHBOARD_PORT=8265 \
+	SQLALCHEMY_DATABASE_URL=sqlite:////tmp/local.db \
+	uv run pytest -o python_files="backend/tests/unit/*/test_*.py"
```

Comment:
@aittalam @veekaybee did we speak about moving the necessary config to the conftest instead for tests????

Reply:
Deferring this conversation to a later cleanup effort :)